deepchem/feat/molecule_featurizers/mat_featurizer.py

```diff
@@ -0,0 +1,92 @@
+from deepchem.feat.base_classes import MolecularFeaturizer
+from deepchem.utils.molecule_feature_utils import one_hot_encode
+try:
+  import logging
```

Comment:
We should move the numpy/pandas/sklearn declarations outside the try/catch block (only optional dependencies go in there)

Reply:
Done! I've removed the entire try-except block and gotten rid of unnecessary dependencies.